compiler/src/dotty/tools/dotc/typer/Namer.scala

```diff
@@ -1062,15 +1062,24 @@ class Namer { typer: Typer =>
           }
         }
 
+        def addWildcardForwardersNamed(name: TermName, span: Span): Unit =
+          List(name, name.toTypeName)
+            .flatMap(path.tpe.memberBasedOnFlags(_, excluded = Private|Given|ConstructorProxy).alternatives)
+            .foreach(addForwarder(name, _, span)) // ignore if any are not added
+
         def addWildcardForwarders(seen: List[TermName], span: Span): Unit =
+          val nonContextual = mutable.HashSet(seen: _*)
           for mbr <- path.tpe.membersBasedOnFlags(required = EmptyFlags, excluded = PrivateOrSynthetic) do
             if !mbr.symbol.isSuperAccessor then
               // Scala 2 superaccessors have neither Synthetic nor Artfact set, so we
               // need to filter them out here (by contrast, Scala 3 superaccessors are Artifacts)
               val alias = mbr.name.toTermName
-              if !seen.contains(alias)
-                && mbr.matchesImportBound(if mbr.symbol.is(Given) then givenBound else wildcardBound)
-              then addForwarder(alias, mbr, span)
+              if mbr.symbol.is(Given) then
```

Comment:
I don't understand the special treatment of `Given` here. I did not see anything in the issue that referred to that. Can you explain what happens here?

Reply:
When we discover a `given` definition and the import selector permits given definitions then here we only export that single definition. But we still want to count the name of that given as unseen, so we may export non-given overloads.

In the case that is not given then we find all overloads that are not given and export those. We then add the name to the set of seen names so that we do not attempt to export the same overload twice, which causes a crash.